features/opennms-es-rest/main-module/src/main/resources/OSGI-INF/blueprint/mainModuleBlueprint.xml

```diff
@@ -29,6 +29,8 @@
       <cm:property name="archiveAlarmChangeEvents" value="true" />
       <cm:property name="archiveOldAlarmValues" value="true" />
       <cm:property name="archiveNewAlarmValues" value="true" />
+      <cm:property name="timeout" value="5000" /> <!-- 5 second timeout for Elasticsearch operations -->
```

Comment:
The timeout now does affect connection timeouts and I've added a ```socketTimeout``` parameter that controls the read timeout (which probably percolates down to SO_TIMEOUT inside httpclient).